sklearn/decomposition/_nmf.py

```diff
@@ -830,10 +858,9 @@ def _fit_multiplicative_update(
 
         # update H
         if update_H:
-            delta_H = _multiplicative_update_h(
-                X, W, H, beta_loss, l1_reg_H, l2_reg_H, gamma
+            H = _multiplicative_update_h(
+                X, W, H, None, None, beta_loss, l1_reg_H, l2_reg_H, gamma, None
             )
```

Comment:
Would using keyword arguments make the code more explicit as `None` is passed? E.g:
```suggestion
            H = _multiplicative_update_h(
                X,
                W,
                H,
                A=None,
                B=None,
                beta_loss=beta_loss,
                l1_reg_H=l1_reg_H,
                l2_reg_H=l2_reg_H,
                gamma=gamma,
                rho=None,
            )
```